### ISSUE TITLE
SOAP API error?

### ISSUE THREAD

**Issue by icarinix** [Inchoo/Inchoo_PHP7#15]
Hi All,

One of my clients uses 'carrytheone' to connect to the Kashflow SaaS app - this is all done via the Magento API. Tonight we attempted to switch over to PHP7 - 'carrytheone' reported that Magento returned and error - Procedure 'login' not present - it didn't provide a stack trace.

This only happened after changeover so seems to be potentially something in the API code not sitting well with PHP7?

Naturally, switched the client back to an older PHP for now.
Mark.

**Comment by ivanweiler** [comment]
Interesting, thx for reporting. If anyone has additional info on the topic, please share ..

**Comment by icarinix** [comment]
Just as an update on this, we've found the issue - for anyone else affected, it seems that switching version to PHP7 from PHP5.x breaks the WSDL cache Magento keeps (likely because of different encodings or some optimisation between versions).

This solution from StackOverflow solved the issue - 
http://stackoverflow.com/questions/12425909/magento-soap-2-api-fatal-error-procedure-login-not-present
Basically cd /tmp then rm wsdl*

Not an issue in the plugin but probably useful to know - I'll close the issue report :)
